en/debug/system_console.md

```diff
@@ -11,9 +11,7 @@ There are multiple shells, but only one console: The system console is the locat
 
 > **Info**
 > USB shell: To just run a few quick commands or test an application connecting to the USB
-> shell is sufficient. The Mavlink shell can be used for this, see below.
-> The hardware serial console is only needed for boot debugging or when USB should be used
-> for MAVLink to connect a [GCS](../qgc/README.md).
+> shell is sufficient. The Mavlink shell can be used for this, see [below](#mavlink_shell).
```

Comment:
MAVLink

Reply:
Done

Reply:
It predates your change, but this tip, and indeed the whole section here (System Console vs. Shell) is unclear.

For the tip,it says "The Mavlink shell can be used for this", but "this" is not defined. I think you are saying that the MAVLink shell can also be used for quick tests?

For the system console vs shell thing I think this is saying that a console is a type of output that is accessed through a shell. You can get this from several different shells - USB, MAVlink whatever. It is also called the NSH console. Is that correct?

If so, would bullet point all the consoles, including MAVLink, then have info at the end explaining what they are good for.

Reply:
> I think you are saying that the MAVLink shell can also be used for quick tests?

Correct.

There are a few differences between the system console and the shell.
System console:
- runs on one specific UART (debug port, as configured in NuttX), used for deeper debugging/devel (bootup, NuttX, startup scripts, board bringup, development on central parts of PX4, such as uORB).
- commonly attached via FTDI cable
- you see the boot output (and all the auto-started tasks)

Whereas the shell is more high-level and can also be used through MAVLink (when there was no mavlink shell yet, it was used directly via USB for example). You'll see the output of what you started, but not of what's already running (but you won't see the output of a task running on the work queue).

Do you want to make a suggestion, so it's understandable for you?

Reply:
I'll commit a change for you to check.

Reply:
Done!